pkg/cmd/admin/router/router.go

```diff
@@ -755,6 +756,10 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 					}
 				}
 			}
+			if len(defaultCert) == 0 {
+				// When a user does not provide the default cert, create one via a Service annotation
+				t.Annotations = map[string]string{"service.alpha.openshift.io/serving-cert-secret-name": name + "-certs"}
```

Comment:
> @deads2k is there a constant for this?

Down in a controller package.  If its going to be used elsewhere, let's go ahead and promote it to a common package.